pyradiosky/skymodel.py

```diff
@@ -261,6 +262,17 @@ def __init__(
             expected_type=int,
             required=False,
         )
+        desc = (
+            "Healpix pixel ordering (ring or nested). Only required for HEALPix maps."
+        )
+        self._hpx_order = UVParameter(
+            "hpx_order",
+            description=desc,
+            value="ring",
+            expected_type=str,
+            required=False,
```

Comment:
I'd vote for making it required, but defaulting it in the way you do below. Unless that breaks pyuvsim in some way.

Reply:
It's pretty common in HEALPix to assume ring ordering by default, since it's much more common. I can make a deprecation warning saying it will be required in the future, but I'm not sure why it should be if other HEALPix packages don't require it.

Reply:
@bhazelton I've made the changes we discussed -- It's now set to required when the component type is healpix; it will default to 'ring' if unset, and the docstring says so.